src/main.c

```diff
@@ -1812,7 +1813,9 @@ static void cmdline_handler(int argc, char **argv)
 #if ENABLE_CONVERT == 1
 		{CONVERT, "convert", "convert <INBUNDLE> <OUTBUNDLE>", "Convert to casync index bundle and store", convert_start, convert_group, FALSE},
 #endif
+#if ENABLE_CHECKSUM == 1
 		{CHECKSUM, "checksum", "checksum <DIRECTORY>", "Deprecated", checksum_start, NULL, FALSE},
```

Comment:
This has been marked as deprecated since 2017. We should probably just remove it completely.

@ejoerns?

Reply:
-@jluebbe Could not find out when we actually decided to mark this deprecated, but it is neither documented nor required and was only useful for the already-removed network mode I guess..

So just remove it.

Reply:
Yeah, I also tried to figure out when and why it was deprecated. If no-one beats me to it, I'll just send a preliminary PR that just removes it, so this one doesn't have to deal with it at all.